api/locales/en/common.json

```diff
@@ -584,11 +584,12 @@
     "adminCommunication": "Administrator communications",
     "adminCommunicationLabel": "Administrator communications",
     "adminCommunicationExplain": "Messages that you share here will be send to all admins or all users of the network",
-    "created": ""
+    "created": "",
```

Comment:
please dont change the common.json in the api... edit on the web and then use the command:
`$ yarn translations` to copy to the api